cypress/e2e/specialFeatures/liteSiteWeight/index.cy.js

```diff
@@ -0,0 +1,102 @@
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+
+import assertPageWeight from './assertions/liteSiteWeight';
+
+const liteTestSuites = [
+  {
+    path: '/hindi.lite?renderer_env=live',
+    service: 'hindi',
+    runforEnv: 'local',
+    pageType: 'Home',
+    tests: [assertPageWeight],
```

Comment:
We could extract this into a variable and then all we need to do is set tests by name in each test suite? 

```
const tests = [assertPageWeight];
```

```suggestion
    tests
```